routes/router.js

```diff
@@ -87,12 +87,10 @@ module.exports = function(app, passport, path) {
 	  	function(req, res) {
 	  		console.log(JSON.stringify(req.user));
 	  		if (!req.user) {
-	  			// res.redirect('/login');
```

Comment:
Why are you deleting your commented out response. Appears as tho you dont res.send to the user ?